worldedit-core/src/main/java/com/boydti/fawe/object/clipboard/DiskOptimizedClipboard.java

```diff
@@ -185,7 +185,7 @@ public boolean setBiome(BlockVector3 position, BiomeType biome) {
 
     @Override
     public boolean setBiome(int x, int y, int z, BiomeType biome) {
-        setBiome(getIndex(x, y, z), biome);
+        setBiome(getIndex(x, 0, z), biome);
```

Comment:
Why was this changed from `y` to `0`?

Reply:
because it's broken otherwise. Clipboards do not have 3D biome support yet, and it was a mistake for the y to be there in the first place (as stated in the commit for the change).